app/grandchallenge/cases/models.py

```diff
@@ -948,3 +980,87 @@ def start_dicom_import_job(self):
             self._mark_failed(
                 error_message="An unexpected error occurred", exc=e
             )
+
+    def get_job_output_manifest(self, *, import_job):
+        output_uri = import_job["outputS3Uri"]
+        parsed = urlparse(output_uri)
+        bucket = parsed.netloc
+        key = parsed.path.lstrip("/") + "job-output-manifest.json"
```

Comment:
You could also just get the bucket name from the settings `settings.AWS_HEALTH_IMAGING_BUCKET_NAME` and then maybe add 2 new environment variables for the 2 prefixes we have in that bucket `logs` and `inputs` and then just construct the key from the those environment variables. For the uploading of the deidentified data to the HI bucket, I was going to do it like that.